docs/csharp/language-reference/operators/member-access-operators.md

```diff
@@ -150,6 +150,8 @@ if (handler != null)
 }
 ```
 
+That is a thread-safe way to ensure that only a non-null `handler` is invoked. Because delegate instances are immutable, no thread can change the value referenced by the `handler` local variable. In particular, if the code executed by another thread unsubscribes from the `PropertyChanged` event and `PropertyChanged` becomes `null` before `handler` is invoked, the value referenced by `handler` remains unaffected.
```

Comment:
I'd add the following to re-introduce the `?.` operator:

```suggestion
That is a thread-safe way to ensure that only a non-null `handler` is invoked. Because delegate instances are immutable, no thread can change the value referenced by the `handler` local variable. In particular, if the code executed by another thread unsubscribes from the `PropertyChanged` event and `PropertyChanged` becomes `null` before `handler` is invoked, the value referenced by `handler` remains unaffected. The `?.` operator evaluates its left operand no more than once, guaranteeing that it cannot be changed to `null` after being verified as non-null.
```

Reply:
Thanks @BillWagner. I've applied the suggestion with the minor update ("left" -> "left-hand")